src/index.html

```diff
@@ -6,8 +6,34 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="container">
+    <div data-qa="card" class="card">
+      <img src="images/imac.jpeg"
+      alt="iMac"
+      class="card__image"
+      >
```

Comment:
Still not fixed from the previous review
```suggestion
      <img
        src="images/imac.jpeg"
        alt="iMac"
        class="card__image"
      >
```